ledger/internal/appcow.go

```diff
@@ -458,10 +459,24 @@ func (cb *roundCowState) StatefulEval(gi int, params *logic.EvalParams, aidx bas
 	calf := cb.child(1)
 	defer calf.recycle()
 
+	var cx *logic.EvalContext
+	defer func() {
+		if err != nil {
+			ledgerAppCallErrTotal.Inc(nil)
+		} else if pass {
+			ledgerAppCallGoodTotal.Inc(nil)
+		} else {
+			ledgerAppCallRejTotal.Inc(nil)
```

Comment:
I think this will count inners too. Is that what you want?

Further, a failed inner will appear as two fails. And a successful inner that executes before a failed inner will get counted as good, even though it ends up going away.

Reply:
Good point, probably need to count both top level contracts and inners separately.
Am I understand it right and the only point where inners can be counted is an inner call inside the AVM evaluator?